libraries/WiFi/src/WiFiMulti.h

```diff
@@ -59,13 +59,14 @@ class WiFiMulti
     bool _bStrict = true;
     bool _bAllowOpenAP = false;
     bool _bTestConnection = false;
-    String _testPhrase = "This is a very simple HTML file.";
+    String _testPhrase = "simple HTML file.";
     String _testURL = "http://www.brainjar.com/java/host/test.html";
```

Comment:
It may be better to default this to www.google.com or similar that returns a very small page. Checking the actual content is not super critical to test that the connection was successful to the remote endpoint, just that a connection to the remote was successful.

Reply:
I was/am concerned that a captive portal would return 200 and fool me.  This test is almost purely to skip captive portals.  I test it with the xfinitywifi AP at the office, but it could be the Starbucks next door.  Thoughts?  I wonder if looking for a 302 header would be better...

Reply:
perhaps checking for a 302 would work as well. The ultimate goal would be to avoid downloading the full response when it is not needed in most cases. Only the first few lines of the response will be needed to detect a redirect or if it is a general 200. Having client.find() will result in a lot of heap usage for the duration of the find and it may not even find the string.

Reply:
will client.find() abort as soon as it finds the target string?  302 Found in this case

Reply:
Make that 200 OK...  I updated the h file.